vendor/bat-native-ads/src/bat/ads/internal/static_values.h

```diff
@@ -44,48 +44,69 @@ const char kDefaultUserModelLanguage[] = "en";
 static const int kDoNotDisturbFromHour = 21;  // 9pm
 static const int kDoNotDisturbToHour = 6;     // 6am
 
-const std::map<std::string, bool> kSupportedRegions = {
-  // {{region, targeted}}
-
-  { "US", true  },  // United States of America
-  { "CA", true  },  // Canada
-  { "GB", true  },  // United Kingdom (Great Britain and Northern Ireland)
-  { "DE", true  },  // Germany
-  { "FR", true  },  // France
-  { "AU", true  },  // Australia
-  { "NZ", true  },  // New Zealand
-  { "IE", true  },  // Ireland
-  { "AR", false },  // Argentina
-  { "AT", false },  // Austria
-  { "BR", false },  // Brazil
-  { "CH", false },  // Switzerland
-  { "CL", false },  // Chile
-  { "CO", false },  // Colombia
-  { "DK", false },  // Denmark
-  { "EC", false },  // Ecuador
-  { "IL", false },  // Israel
-  { "IN", false },  // India
-  { "IT", false },  // Italy
-  { "JP", false },  // Japan
-  { "KR", false },  // Korea
-  { "MX", false },  // Mexico
-  { "NL", false },  // Netherlands
-  { "PE", false },  // Peru
-  { "PH", false },  // Philippines
-  { "PL", false },  // Poland
-  { "SE", false },  // Sweden
-  { "SG", false },  // Singapore
-  { "VE", false },  // Venezuela
-  { "ZA", false },  // South Africa
-  { "KY", true  }   // Cayman Islands
+const std::map<int, std::map<std::string, bool>> kSupportedRegionsSchemas = {
```

Comment:
I would do unsigned int

Reply:
@SergeyZhukovsky `PrefRegistry` does not have unsigned versions of `RegisterIntegerPref` and `SetInteger`, see https://chromium.googlesource.com/chromium/src.git/+/72.0.3626.80/components/prefs/pref_registry_simple.h?autodive=0%2F%2F%2F

Reply:
sounds good